app/facades/attendance_show_facade.rb

```diff
@@ -9,24 +9,41 @@ def initialize(attendance, updatable: false)
   end
   
   def alias_options_for(student)
-    @attendance.turing_module.unclaimed_aliases.sort_by do |zoom_alias|
+    @attendance.turing_module.unclaimed_aliases(@attendance.meeting_id).sort_by do |zoom_alias|
       -1 * string_distance(student.name, zoom_alias.name)
     end.map do |zoom_alias|
       [zoom_alias.name, zoom_alias.id]
     end
   end
 
+  def unassigned_zoom_aliases
+    @attendance.turing_module.unclaimed_aliases(@attendance.meeting_id).map { |zoom_alias| zoom_alias.name }.sort
+  end
+
   def student_attendances
     @attendance.student_attendances.includes(:student).by_attendance_status
   end
 
+  def instructor_zoom_aliases
```

Comment:
I really like creating separate methods for regular unassigned aliases compared to instructor zoom aliases in order to easily separate what is needed in the view.

Reply:
Thank you!